docs/source/getting-started/code/ui-after/MainView.tsx

```diff
@@ -4,7 +4,7 @@
 import React, { useMemo } from 'react';
 import { Container, Grid, Header, Icon, Segment, Divider } from 'semantic-ui-react';
 import { Party } from '@daml/types';
-import { User } from '@daml-ts/create-daml-app-0.1.0/lib/User';
+import { User } from '@daml.js/create-daml-app-0.1.0/lib/User';
```

Comment:
@rohanjr We need to fix this import in the same way as the other imports yesterday. Will this be fixed by the PR to avoid duplicating files?

Reply:
This is in the "after" section, so it's not in the template and files will still be duplicated for now. But I did go through and update the parts that are actually included in the guide (which this part isn't). It's fiddly and error-prone now I know.